signalfx-async-wrapper.js

```diff
@@ -12,67 +14,87 @@ class SignalFxWrapper {
     originalContext,
     originalCallback,
     dimensions,
-    accessToken
+    disableTracing,
+    disableMetrics
   ) {
     this.originalObj = originalObj;
     this.originalFn = originalFn;
     this.originalEvent = originalEvent;
     this.originalContext = originalContext;
     this.originalCallback = originalCallback;
+    this.enableMetrics = sfxHelper.isMetricsEnabled(disableMetrics);
+    this.enableTracing = sfxHelper.isTracingEnabled(disableTracing);
 
-    sfxHelper.setAccessToken(accessToken);
-    sfxHelper.setLambdaFunctionContext(this.originalContext, dimensions);
-    sfxHelper.sendCounter('function.invocations', 1);
+    this.metricSender = sfxHelper;
+    if (!this.enableMetrics) {
+      this.metricSender = noopMetricSender;
+    }
+
+    this.execMeta = sfxHelper.getExecutionMetadata(this.originalContext);
+    sfxHelper.setDefaultDimensions(dimensions, this.execMeta);
+
+    this.metricSender.sendCounter("function.invocations", 1);
```

Comment:
we should list somewhere these "generic" serverless metrics

Reply:
Do you mean document them somewhere? For internal eng docs or product docs?

Reply:
Both, of course, this PR doesn't need to wait for that.